.github/workflows/repair-1266-once.yml

```
@@ -0,0 +1,59 @@
+          toolchain: 1.95.0
+          components: rustfmt,clippy
+
+      - uses: Swatinem/rust-cache@v2
+        with:
+          cache-on-failure: true
+
+      - name: Apply, verify, and publish the clean repair commit
+        env:
+          EXPECTED_TEMP_HEAD: ${{ github.sha }}
+          GH_TOKEN: ${{ github.token }}
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:24:75: Double quote to prevent globbing and word splitting

(shellcheck)